google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
```

Comment:
If you guys feel very strongly about this, I almost prefer to try and get rid of APP_ENGINE_FLEX_COMPAT instead of renaming the class. Following Etan's suggestion, it'd be weird to be comparing e.g., targetEnvironments to project types.
I'm personally OK with things this way, though.